pkgs/applications/office/sozi/default.nix

```
@@ -0,0 +1,38 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "sozi";
+  version = "23.7.25-1690311612";
+
+  src = fetchurl {
+    url = "https://github.com/sozi-projects/Sozi/releases/download/v23.07/Sozi-23.7.25-1690311612.AppImage";
```

The `version` can be re-used here, which makes updating future versions of this package easier and less error-prone:
```suggestion
    url = "https://github.com/sozi-projects/Sozi/releases/download/v${lib.versions.majorMinor version}/Sozi-${version}.AppImage";
```